v2/cmd/controller/app/setup.go

```diff
@@ -314,6 +316,19 @@ func getDefaultAzureTokenCredential(cfg config.Values, setupLog logr.Logger) (az
 		return credential, nil
 	}
 
+	if userAssignedCredentials := os.Getenv(common.AzureUserAssignedIdentityCredentials); userAssignedCredentials != "" {
+		options := azcore.ClientOptions{
+			Cloud: parseCloudTypeFromEnvironment(),
+		}
+
+		credential, err := dataplane.NewUserAssignedIdentityCredential(context.Background(), userAssignedCredentials, dataplane.WithClientOpts(options))
```

Comment:
`userAssignedCredentials` is actually a _path_ to credentials (mounted in the pod?)

 Can you call the variable something like `xyzPath`, to make that clear?

Reply:
Yeah its a path to the credentials file. I renamed it.